### ISSUE TITLE
Source code in Script tag is html encoded

### ISSUE THREAD

**Issue by bcpeinhardt** [nakaibuild/nakai#26]
When I try to add the following in a `Script` tag
```js
Script("
        document.addEventListener(\"DOMContentLoaded\", (event) => {
            document.body.addEventListener('htmx:beforeSwap', function(evt) {
                if (evt.detail.xhr.status === 422) {
                    // allow 422 responses to swap as we are using this as a signal that
                    // a form was submitted with bad data and want to rerender with the
                    // errors
                    //
                    // set isError to false to avoid error logging in console
                    evt.detail.shouldSwap = true;
                    evt.detail.isError = false;
                }
            });
        })
      ")
```

The `>` is being sent as `&gt;`

**Comment by aslilac** [comment]
I thought I'd tested this and it was fine. it's escaped, but does the browser interpret it correctly?

**Comment by bcpeinhardt** [comment]
Should it be escaped in JS source? I didn't think it should be. There's an error in the console:
`Uncaught SyntaxError: expected expression, got '&'`
and the source comes through as 
```js
document.addEventListener("DOMContentLoaded", (event) =&gt; {
            ... all the rest of it
        })
```
which I presume is why it's not working

**Comment by aslilac** [comment]
escaping stuff in HTML is weird and janky. 😵‍💫 this shouldn't be hard to fix tho.

**Comment by aslilac** [comment]
Fixed in 0.11

**Comment by bcpeinhardt** [comment]
Hahaha I had just sat down to do it, thank you!
